components/Form/FormInput.js

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+import { FormLabel } from './FormLabel';
+
+const regex = /^[a-zA-Zá-žÁ-Ž0-9 .]+$/;
+
+export const FormInput = ({
+  defaultValue,
+  onChange,
+  labelClassName,
+  label,
+  required,
+  value,
+  pattern,
+  className,
+  ...props
+}) => {
+  const [userInput, setUserInput] = useState(defaultValue);
+  const [isValid, setIsValid] = useState(true);
+
+  const handleInputChange = (e) => {
+    if (onChange) {
+      onChange(e);
+    } else {
+      if (pattern) {
+        if (e.target.value) {
+          setIsValid(regex.test(e.target.value));
+        }
+      }
+      setUserInput(e.target.value);
```

Comment:
```suggestion
        if (pattern && e.target.value) {
          setIsValid(regex.test(e.target.value));
        }
      
      setUserInput(e.target.value);
```

No need for that nested IF. Also, it would be nice if that `setUserInput` is separated with a new line. I visually like, when there is a new line after IF

Reply:
Please apply to all places where this pattern code is